docs/UserGuide.md

```diff
@@ -433,33 +433,38 @@ Examples:
 ContaX contacts and appointments data are saved in the hard disk automatically after any command that changes contact data in JSON format at `[JAR file location]/data/addressbook.json` and `[JAR file location]/data/schedule.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, ContaX will discard all data and start with an empty data file at the next run.
+If your changes to the data file makes the format invalid, ContaX wil try to read valid objects and import, and will skip objects with attributes that have invalid formats (e.g. having alphabets in a Phone number), and any duplicate Persons. 
+
+However, if the JSON formatting is broken, a blank AddressBook/Schedule will be loaded instead.
+
+Refer to the [FAQ](#faq) for more details
 </div>
 
 ### Importing data
 
 In order to import data from external sources into ContaX, there are the following options:
 
 * Replacing `addressbook.json` or `schedule.json` file with a valid ContaX formatted generated by another system with ContaX
-* [Simple CSV import (GUI-based option)](#simple-csv-import)
-* [Advanced CSV import (Command based)](#advanced-csv-import--importcsv)
+* [Importing CSV Files](##importing-csv-files-importcsv)
 
 ### Importing CSV Files: `importcsv`
 
-Format: `importcsv f/FILEPATH [n/COLUMNNUM] [p/COLUMN_PERSON] [e/COLUMN_EMAIL] [a/COLUMN_ADDRESS] [t/COLUMN_TAGS]`
+Format: `importcsv f/FILEPATH [n/COLUMN_NAME] [p/COLUMN_PHONE] [e/COLUMN_EMAIL] [a/COLUMN_ADDRESS] [t/COLUMN_TAGS]`
 
 * Imported contacts will be appended to the current address book.
 * Each argument represents the corresponding attribute for a person.
-  * n = Name
-  * p = Phone
-  * e = Email
-  * a = Address
-  * t = Tagged
+  * COLUMN_NAME - specifies column number for Name
+  * COLUMN_PHONE - specifies column number for Phone
+  * COLUMN_EMAIL - specifies column number for Email
+  * COLUMN_ADDRESS - specifies column number for Address
+  * COLUMN_TAGS - specifies column number for Tags
```

Comment:
I think monospaced formatting on the arguments will make them easier to read.

Reply:
thanks for the catch, updated in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/251/commits/04af621a4f32d0c84a6557320113901da95ef91b